src/wwwroot/web.config

```diff
@@ -0,0 +1,9 @@
+<configuration>
+    <system.webServer>
+        <staticContent>
+            <mimeMap fileExtension=".otf" mimeType="font/opentype" />
+            <mimeMap fileExtension=".woff" mimeType="font/woff" />
+            <mimeMap fileExtension=".woff2" mimeType="font/woff2" />
```

Comment:
You have missed `.ttf` and `.eot`, it seems to be an important issue (not like indentation), please fix it.